GPU/Common/TextureCacheCommon.cpp

```diff
@@ -751,20 +757,20 @@ void TextureCacheCommon::DetachFramebuffer(TexCacheEntry *entry, u32 address, Vi
 	}
 }
 
-bool TextureCacheCommon::AttachFramebuffer(TexCacheEntry *entry, u32 address, VirtualFramebuffer *framebuffer, u32 texaddrOffset) {
+bool TextureCacheCommon::AttachFramebuffer(TexCacheEntry *entry, u32 address, VirtualFramebuffer *framebuffer, u32 texaddrOffset, FramebufferNotificationChannel channel) {
 	static const u32 MAX_SUBAREA_Y_OFFSET_SAFE = 32;
 
 	AttachedFramebufferInfo fbInfo = { 0 };
 
 	const u32 mirrorMask = 0x00600000;
 	u32 addr = address & 0x3FFFFFFF;
-	u32 texaddr = entry->addr + texaddrOffset;
+	u32 texaddr = (entry->addr + texaddrOffset) & ~mirrorMask;
```

Comment:
Wait, I thought we moved this into the if below specifically for RAM framebuffer copies?  Should work for depth too, no?  See 15f2e48887de92cf3ed61dd2aab212bb0d24a0a6.

Although I think the check below probably should be more like `(entry->addr & 0x3F000000) == 0x04000000`.  I guess that part shouldn't matter, in practice.

-[Unknown]

Reply:
Woops, this branch was kind of old when I resurrected it so some misplaced stuff like this has survived.